models/webhook.go

```diff
@@ -629,10 +665,38 @@ func (h HookEventType) Event() string {
 		return "repository"
 	case HookEventRelease:
 		return "release"
+	case HookEventOrg:
+		return "organization"
+	case HookEventTeam:
+		return "team"
+	case HookEventTeamMember:
+		return "membership"
 	}
 	return ""
 }
 
+// HookEventLevel leve of a hook event
+type HookEventLevel int64
+
+const (
+	// HookEventLevelRepo all hook types can be used
+	HookEventLevelRepo HookEventLevel = iota
```

Comment:
```suggestion
  HookEventLevelRepo HookEventLevel = iota + 1
```

iota along makes it zero and it feels kind of weird seeing a valid state as 0